gazebo_ros2_control_demos/examples/example_gripper.cpp

```diff
@@ -0,0 +1,58 @@
+// Copyright 2022 Stogl Robotics Consulting UG (haftungsbeschränkt)
```

Comment:
do you mind to add yourself as an autor in the file but use the open robotics copyright ?

Reply:
Sorry I was applying the Ignition review guidelines, it's actually not an issue here. In Ignition we ask that people voluntarily assign their copyright to OSRF which allows the Foundation more flexibility in the future.

Anyhow @tfoote or @chapulina can add here more details

Reply:
OK, I'll do this for Ignition plugin